autonomy/chain/mint.py

```diff
@@ -86,7 +86,7 @@ def mint_component(
     """Publish component on-chain."""
 
     if dependencies is not None:
-        dependencies = sorted(dependencies)
+        dependencies = sorted(set(dependencies))
```

Comment:
add `set` also for agents

Reply:
Right now we only support one agent per service so no need there